source/rust_verify_test/tests/cell_lib.rs

```diff
@@ -289,7 +289,7 @@ test_verify_one_file! {
         pub fn foo(r: cell::PointsTo<Foo>) {
             f(r);
         }
-    } => Err(e) => assert_error_msg(e, "the trait `std::marker::Sync` is not implemented for `Foo`")
+    } => Err(e) => assert_rust_error_code(e, "E0277")
```

Comment:
I mentioned this in a different comment, but I like including the string "the trait `std::marker::Sync` is not implemented for `Foo`" in the error assertion.